src/api/spec/controllers/webui/package_controller_spec.rb

```diff
@@ -207,7 +207,7 @@
       end
 
       it { expect(controller).to set_flash[:error] }
-      it { expect(response).to redirect_to(package_edit_path(project: source_project, package: source_package)) }
+      it { expect(response).to redirect_to(package_show_path(project: source_project, package: source_package)) }
```

Comment:
The commit description is outdated, apart from that  great job :clap:

Reply:
Oops, I forgot to update it!